### ISSUE TITLE
Fix error that `#jsx-fragment` is invalid element

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#641]
- Markuplint version: `3.0.0`
- Parser lang: `jsx-parser`
- Node.js version: `18.12.1`
- OS: macOS

## Describe the bug

```
The #jsx-fragment is invalid element (x:x): Broke mapping nodes
```

## Code Example

```jsx
<p>{array.map(_ => <></>)}</p>
```

The cause may is call `querySelectorAll` in `disallowed-element` rule according to research.

## Steps To Reproduce

1. Specify `@markuplint/jsx-parser`
2. Specify `disallowed-element` rule
3. Run

## The current behavior

```
The #jsx-fragment is invalid element (x:x): Broke mapping nodes
```

## The expected behavior

No warning
